spec/features/application/web_server_spec.rb

```diff
@@ -33,4 +17,37 @@
 
     expect(page.status_code).to eq 403
   end
+
+  scenario 'content security policy' do
+    visit_200_page '/home'
+
+    expect(response_headers['Content-Security-Policy']).to eq [
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.